### PR TITLE
Create + use actinia processes dict with only one returns

### PR DESCRIPTION
**Pull request by mmacata**
While GRASS GIS modules can have multiple outputs, the openEO API allows only one. This PR duplicates the ACTINIA_PROCESS_DESCRIPTION_DICT and manipulates the copy to split up modules with multiple outputs. Instead of an array of outputs, the module then has only one output object.

Example: 
`r.slope.aspect` will not appear as a single module but as many for each of its outputs:

```
{
	"id": "r.slope.aspect_slope",
	"parameters": [...],
	"returns": {
		"description": "Name for output slope raster map. ",
		"name": "slope",
		"optional": true,
		"schema": {
			"subtype": "cell",
			"type": "string"
		}
	}
},
{
	"id": "r.slope.aspect_aspect",
	"parameters": [...],
	"returns": {
		"description": "Name for output aspect raster map. ",
		"name": "aspect",
		"optional": true,
		"schema": {
			"subtype": "cell",
			"type": "string"
		}
	}
},
...
```

Hint:
To see this change in action, the actinia module filters need to be deployed in the used actinia instance. The current configured instance https://actinia-dev.mundialis.de is not yet up-to-date. These changes won't break things though.

For each touched file the lint errors were fixed.

### CODE REVIEW

**Comment by huhabla** [comment]
So openEO API still allows only a single output for a process?

**Comment by mmacata** [comment]
As per API specification `returns` is an object with `description` and `schema` and not an array of objects - yes, only a single output for predefined processes is allowed.

File: src/openeo_grass_gis_driver/register_actinia_processes.py [hunk 1]
```diff
@@ -17,18 +18,33 @@ def register_processes():
...
+                    pm['returns'] = returns
+                    process = pm["id"].replace('.', '_')
+                    ACTINIA_OPENEO_PROCESS_DESCRIPTION_DICT[process] = pm
+
+            else:
+                # if no output, assign empty object
+                module['returns'] = {}
+                ACTINIA_OPENEO_PROCESS_DESCRIPTION_DICT[process] = module
+
+        print("... successfully registered modules!")
```
Comment (metzm) [inline src/openeo_grass_gis_driver/register_actinia_processes.py:50]: Is this `print()` left-over debugging code?

**Review by metzm — CHANGES_REQUESTED** [review]
Looks good to me, but I guess the new `print()` line needs to be removed before merging.

**Review by metzm — APPROVED** [review]
Thanks for the explanation, ready to merge.

**Pull request merged**